packages/components/src/highlight-cards/mobile-highlight-cards.tsx

```diff
@@ -8,16 +8,31 @@ type MobileHighlightCardProps = {
 	count: number | null;
 	previousCount?: number | null;
 	icon?: React.ReactElement;
+	preformattedValue?: string;
 };
 
-function MobileHighlightCard( { heading, count, previousCount, icon }: MobileHighlightCardProps ) {
-	// We require at minimum a heading and a count.
-	const isValidHighlight = count !== null && heading.length > 0;
-	if ( ! isValidHighlight ) {
+function MobileHighlightCard( {
+	heading,
+	count,
+	previousCount,
+	icon,
+	preformattedValue,
+}: MobileHighlightCardProps ) {
+	// We'll accept a count or a preformatted string as our value.
+	// If both are provided, we'll use the preformatted string for display.
+	const hasValidDisplayValue =
+		( count !== null && count !== undefined ) ||
+		( preformattedValue !== undefined && preformattedValue.length > 0 );
```

Comment:
I know this might be more concise but I usually only do like  `if(performattedValue)`, [which checks for undefined, null as well as empty string etc](https://developer.mozilla.org/en-US/docs/Glossary/Truthy), which is better for the bundle size. There are some cases below as well.